lms/templates/header/header.html

```diff
@@ -14,7 +14,7 @@
 # App that handles subdomain specific branding
 from branding import api as branding_api
 from openedx.core.djangoapps.lang_pref.api import header_language_selector_is_enabled, released_languages
-
+from student.models import UserProfile
```

Comment:
Nit, but can we leave the newline at the bottom? below this import?

It'll shrink the diff ;)